modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -124,15 +124,16 @@ object GlobalSnapshotConsensusFunctions {
           maybeTipTrigger.fold(lastGS.height)(_.height),
           maybeTipTrigger.fold(lastGS.subHeight.next)(_ => SubHeight.MinValue),
           lastGSHash,
-          blocksInRange,
+          blocksInRange.map(BlockAsActiveTip(_, 0L)),
```

Comment:
Is that always true? I guess not all new blocks will have 0 usages. If that is here just for now and will be changed in the next PR, that's fine.

Reply:
Yes, that's here only temporarily so that the whole change compiles.